docs/user-guide/openai-api/translation-agent.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 5
+---
+
+# Translation Agent on LlamaEdge
+
+This LLM Translation Agent originally built by [Prof. Andrew Ng](https://www.linkedin.com/posts/andrewyng_github-andrewyngtranslation-agent-activity-7206347897938866176-5tDJ/) is designed to facilitate accurate and efficient translation across multiple languages. It employs open source LLMs (Large Language Models) to provide high-quality translations. You can use your own fine-tuned models or any LLMs on Hugging Face like Meta's Llama 3. This documentation shows how the Transgenic Agent utilizes the Gemma-2-9B model for translation.
+
+
+>For commands on starting and running this agent, refer to [GitHub - Second State/translation-agent](https://github.com/second-state/translation-agent/blob/use_llamaedge/step-by-step-use-LocalAI.md).
+
+
+## Run Gemma-2-9B on your own device
```

Comment:
Please follow other articles as an example to describe how to run an open source LLM locally.  Thanks. https://llamaedge.com/docs/user-guide/openai-api/lobechat#prerequisites